### PR TITLE
Fix http constants and status codes

### LINKED ISSUE #6: Wrong status code on not found
You can reproduce it like this:

```bash
$ curl -v http://localhost:8000/ns/users/1
*   Trying ::1:8000...
* Connected to localhost (::1) port 8000 (#0)
> GET /ns/users/1 HTTP/1.1
> Host: localhost:8000
> User-Agent: curl/7.77.0
> Accept: */*
> 
* Mark bundle as not supporting multiuse
< HTTP/1.1 400 Bad Request
< Content-Type: application/json
< Vary: Origin
< Date: Thu, 18 Nov 2021 10:58:22 GMT
< Content-Length: 22
< 
* Connection #0 to host localhost left intact
{"error": "not found"}
```

Need to change status to 404.

### PR DESCRIPTION
**Pull request by ehsundar**
- method names
- status codes
- error messages

closes #6

### CODE REVIEW

**Comment by ehsundar** [comment]
#6

**Comment by ehsundar** [comment]
I plan to refactor your storage impls. I think its better to have the interface def under database package. Also each implementation must move to its own package with specialized constructor

File: database/fs/fs.go [hunk 1]
```diff
@@ -0,0 +1,113 @@
+package fs
```
Comment (rehacktive) [inline database/fs/fs.go:1]: I would keep this in the same package (in the end it's just another "database" implementation)

**Review by rehacktive — CHANGES_REQUESTED** [review]

**Comment by rehacktive** [comment]
> I also would like to refactor storage impls. I think its better to have the interface def under database package. Also each implementation must move to its own package with specialized constructor

having a common interface with no specific constructor makes the code simple. this can be changed when specifically necessary.

**Comment by rehacktive** [comment]
hello - could you please rebase and make the small change? I'll be happy to merge then. thanks @ehsundar

**Comment by ehsundar** [comment]
Hey. I'll consider the requested changes today.

**Review by rehacktive — APPROVED** [review]
all good :) thanks a lot!

**Pull request merged**